client/state/help/directly/reducer.js

```diff
@@ -15,12 +16,23 @@ import {
 	STATUS_ERROR,
 } from './constants';
 
+export const questionAsked = ( state = null, action ) => {
```

Comment:
I've actually been advised against using `createReducer` and `createSelector` unless you need the features they provide. So my default right now is to avoid them unless I know they'll be really useful for the circumstance.